pom.xml

```diff
@@ -139,6 +139,14 @@
       <artifactId>junit-jupiter</artifactId>
       <version>${junit.jupiter.version}</version>
     </dependency>
+
+    <dependency>
```

Comment:
A lot of test class names have changed, so you'll need to update the references and links to them in the README